Makefile

```
@@ -56,17 +56,18 @@ compile_tests: copy_tests
 		test=$${test##*/}; test=$${test%.*}; \
 		$(RISCV_PREFIX)-gcc -c $(CFLAGS) -Iut/riscv-tests/env/p -Iut/riscv-tests/isa/macros/scalar -Iut/riscv-tests/isa/rv32ui -o tests-build/$$test.o tests/$$test.S; \
 		$(RISCV_PREFIX)-gcc -o tests-build/$$test.elf $(LDFLAGS) tests-build/$$test.o sw/mtvec_handler.o; \
-		$(RISCV_PREFIX)-objcopy -j .text -j .data -j .rodata -O binary tests-build/$$test.elf tests-build/$$test.bin; \
-		hexdump -v -e '1/4 "%08x\n"' tests-build/$$test.bin > tests-build/$$test.hex; \
+		$(RISCV_PREFIX)-objcopy -j .text -j .data -j .rodata -O verilog --verilog-data-width=4 tests-build/$$test.elf tests-build/$$test.mem; \
 	done
 
 run_vivado: compile_tests
```

checkmake (0.3.2)

[warning] Target body for "run_vivado" exceeds allowed length of 5 lines (19).

(maxbodylength)